bindings/pyroot/pythonizations/test/rdataframe_asnumpy.py

```diff
@@ -90,10 +90,25 @@ def test_branch_bool(self):
         Test bool data-type as a special case since we cannot adopt
         the std::vector<bool> with numpy arrays
         """
-        df = ROOT.RDataFrame(2).Define("x", "bool(rdfentry_)")
-        npy = df.AsNumpy()
-        self.assertFalse(bool(npy["x"][0]))
-        self.assertTrue(bool(npy["x"][1]))
+        try:
+            treename = "test_branch_bool"
+            filename = "test_branch_bool.root"
+            # Snapshot a TTree so that column 'x' will be of type 'Bool_t'
+            ROOT.RDataFrame(2).Define("x", "bool(rdfentry_)").Snapshot(treename, filename)
+            # The column 'y' will instead have type 'bool'
+            df = ROOT.RDataFrame(treename, filename).Define("y", "bool(rdfentry_)")
+            self.assertEqual(df.GetColumnType("x"), "Bool_t")
+            self.assertEqual(df.GetColumnType("y"), "bool")
+            npy = df.AsNumpy()
+            # Both numpy arrays should have dtype bool
+            self.assertEqual(npy["x"].dtype, bool)
+            self.assertEqual(npy["y"].dtype, bool)
+            self.assertFalse(npy["x"][0])
+            self.assertTrue(npy["x"][1])
+            self.assertFalse(npy["y"][0])
+            self.assertTrue(npy["y"][1])
+        finally:
+            os.remove(filename)
```

Comment:
This results in test failures on Windows:
```txt
2025-10-24T20:22:30.9438074Z ----------------------------------------------------------------------
2025-10-24T20:22:30.9438634Z Traceback (most recent call last):
2025-10-24T20:22:30.9439374Z   File "C:\ROOT-CI\src\bindings\pyroot\pythonizations\test\rdataframe_asnumpy.py", line 111, in test_branch_bool
2025-10-24T20:22:30.9440075Z     os.remove(filename)
2025-10-24T20:22:30.9440834Z PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'test_branch_bool.root'
```

There are many other tests that don't clean up temporary files that they produce either. So IMHO it's fine to fix the test on Windows by just keeping the file around.